src/clang.rs

```diff
@@ -187,7 +187,7 @@ impl Cursor {
 
     /// Is the referent a template specialization?
     pub fn is_template(&self) -> bool {
-        self.specialized().is_valid()
+        self.specialized().map_or(false, |c| c.is_valid())
```

Comment:
This can be `self.specialized().is_some()` now, given we check if the cursor is valid in `specialized`.